auth/service.bal

```diff
@@ -0,0 +1,101 @@
+import ballerina/http;
+import ballerina/crypto;
+import ballerina/uuid;
+import e_backend.data;
+import ballerina/persist;
+import ballerina/sql;
+
+listener http:Listener voterListener = new (9090);
+
+final data:Client dbClient = check new ();
+
+
+public type VoterLogin record { 
+    string nationalId; 
+    string password; 
+};
+
+service /Voter\-registration/api/v1 on voterListener {
```

Comment:
Shall we rename this to `voter-registration` with a lower-case `v`? We usually have lower case letters for services/endpoints.